packages/bot-skeleton/src/services/api/api-base.js

```diff
@@ -63,33 +63,31 @@ class APIBase {
         }
     };
 
-    authorizeAndSubscribe() {
+    async authorizeAndSubscribe() {
         const { token, account_id } = getToken();
         if (token) {
             this.token = token;
             this.account_id = account_id;
             this.api.authorize(this.token);
-            this.api
-                .expectResponse('authorize')
-                .then(({ authorize }) => {
-                    if (this.has_activeSymbols) {
-                        this.toggleRunButton(false);
-                    } else {
-                        this.getActiveSymbols();
-                    }
-                    this.subscribe();
-                    this.account_info = authorize;
-                })
-                .catch(e => {
-                    globalObserver.emit('Error', e);
-                });
+            try {
+                const { authorize } = await this.api.expectResponse('authorize');
+                if (this.has_activeSymbols) {
+                    this.toggleRunButton(false);
+                } else {
+                    this.getActiveSymbols();
+                }
+                await this.subscribe();
```

Comment:
@vinu-deriv looks like nothing changed here and this `subscribe` was not called before `authorize`.

Reply:
yeah, nothing changes there, it is only that since subscribe is async function did not wanted the execution to go further until all three method calls inside  subscribe is resolved

Reply:
@vinu-deriv Yeah, I understand this, but you are trying to handle `AuthorizationRequired` error and in this PR you didn't change anything related to `authorize` because this part of code is working the same, you still make subscribe after successful `authorize`. I don't see how it can fix the error... It's just my thoughts, you can ignore it :)

Reply:
yeah, very true, even I doubt if it will wok, just trial and error since we are not able to replicate the issue in local by any way, so just trying to execute the function in synchronous manner after authorize.